test/witan/app/handler_test.clj

```diff
@@ -4,10 +4,42 @@
             [ring.util.io :refer [string-input-stream]]
             [witan.app.handler :refer :all]
             [witan.app.user :as user]
+            [witan.app.forecast :as forecast]
             [clojure.data.json :as json]))
 
 (def user-id (java.util.UUID/randomUUID))
 
+(defn get-dummy-forecasts []
+  '({:description "Description of my forecast",
+     :name "My Forecast 1",
+     :descendant_id nil,
+     :created #inst "2015-10-06T12:44:17.176-00:00",
+     :series_id #uuid "ca7928d8-ea7d-4bdb-ab16-4c6ae8912830",
+     :in_progress false,
+     :id #uuid "fd44474d-e0f8-4713-bacf-299e503e4f30",
+     :version 0,
+     :owner #uuid "cac4ba3a-07c8-4e79-9ae0-d97317bb0d45"}
+    {:description "Description of my forecast",
+     :name "My Forecast 2",
+     :descendant_id nil,
+     :created #inst "2015-10-06T12:44:17.210-00:00",
+     :series_id #uuid "768f40f8-cf06-4da6-8b98-5227034f7dd5",
+     :in_progress false,
+     :id #uuid "102fef0c-aa17-41bc-9f4e-cc11d18d7ae5",
+     :version 0,
+     :owner #uuid "6961ed51-e1d6-4890-b102-ab862893e3ba"}
+    {:description "Description of my forecast",
+     :name "My Forecast 3",
+     :descendant_id nil,
+     :created #inst "2015-10-06T12:44:17.240-00:00",
+     :series_id #uuid "197481d6-df2a-4175-a288-d596a9709322",
+     :in_progress false,
+     :id #uuid "7185c4e4-739e-4eb8-8e37-f3f4b618ac1d",
+     :version 0,
+     :owner #uuid "cac4ba3a-07c8-4e79-9ae0-d97317bb0d45"}))
+
+(defn auth-header [token] {"Authorization" (str "Token " token)})
```

Comment:
nice one, was thinking the same. maybe we can abstract it even more later (between all routes that have to be authenticated and not, like buddy examples)